README.md

```diff
@@ -33,3 +33,4 @@ Feel free to add your username here :)
 * [MauricioLanda](https://github.com/MauricioLanda)
 * [ShannonMCook](https://github.com/shannonmcook)
 
```

Comment:
Why have your name spaced out from everyone else?;) If you were trying to avoid a merge conflict, then good thinking, but that just takes away all the fun of conflict management ;)
```suggestion
* [Lashuk1729](https://github.com/Lashuk1729)

```